js/constants.js

```diff
@@ -261,3 +261,9 @@ export const JOIN_POST_URL = `${BASE_URL}/users/self/intro`;
 export const SELF_URL = `${BASE_URL}/users/self`;
 export const GITHUB_OAUTH =
   'https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97';
+export const SIGNUP = 'signup';
+export const HOME = 'home';
```

Comment:
we have a HOME_URL, do we need these home and signup?

Reply:
For yes for checking the conditions